tensorflow/tools/ci_build/builds/configured

```diff
@@ -28,6 +28,10 @@ set -e
 CONTAINER_TYPE=$( echo "$1" | tr '[:upper:]' '[:lower:]' )
 shift 1
 
+# Enable support for MKL
+export TF_NEED_MKL=1
```

Comment:
`configured` calls `configure` under the hood, with all-default answers. So I think you can just remove this line and the line below.